docs/source/tutorial_basis_generation.md

```diff
@@ -784,26 +778,57 @@ weak_greedy_basis = reductor.bases['RB']
 Let's see, how the weak-greedy basis performs:
 
 ```{code-cell}
-weak_greedy_errors = compute_proj_errors_orth_basis(weak_greedy_basis, V, fom.h1_0_semi_product)
+weak_greedy_errors = compute_proj_errors_orth_basis(weak_greedy_basis, training_data, fom.h1_0_semi_product)
+
+plt.figure()
+plt.semilogy(trivial_errors, label='trivial')
+plt.semilogy(greedy_errors, label='greedy')
+plt.semilogy(pod_errors, label='POD')
+plt.semilogy(weak_greedy_errors, label='weak greedy')
+plt.ylim(1e-7, 1e2)
+plt.legend()
+plt.show()
+```
+
+We see that the weak-greedy basis is worse than all the other bases!
+How can that be?
+Remember that the weak-greedy basis was trained for a new training data set of 1000 parameter values
+and not the original much smaller training data set.
+So what we actually see here is that the other bases are very good at approximating this small training
+data set, but likely do not generalize well to the entire solution manifold
+
+```{math}
+\{u(\mu) \,|\, \mu \in \mathcal{P}\}.
```

Comment:
How about introducing the solution manifold already at the beginning of the tutorial where the Kolmogorov $N$-width is discussed? Here it looks a bit lost...

Reply:
Done.